terraform/main.tf

```diff
@@ -40,13 +40,18 @@ resource "kubernetes_secret" "baba_service_account_credentials" {
   }
 }
 
-resource "kubernetes_secret" "ror-baba-db-password" {
+resource "kubernetes_secret" "ror-baba-secret" {
   metadata {
-    name      = "${var.labels.team}-${var.labels.app}-db-password"
+    name      = "${var.labels.team}-${var.labels.app}-secret"
```

Comment:
secret or secrets?

Reply:
secret without 's': this is the Kubernetes type  found in the yaml config:
apiVersion: v1
kind: Secret

Reply:
I mean the name e.g. baba-secrets (since it contains multiple passwords/secrets